media-libs/sdl2_mixer/sdl2_mixer-2.0.1.recipe

```diff
@@ -23,9 +23,10 @@ PROVIDES="
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
-	lib:libSDL2_2.0$secondaryArchSuffix
+	lib:libgl$secondaryArchSuffix
+	lib:libsdl2_2.0$secondaryArchSuffix
 	lib:libogg$secondaryArchSuffix
-	lib:libFLAC$secondaryArchSuffix
+	lib:libflac$secondaryArchSuffix
```

Comment:
> I do understand that all this does not make much difference, but I was wondering if there was a recommended or prefered spelling for lib: and devel: items for libs that have uppercase letters in their names.

HaikuPorter uses lowercase internally, but it's preferable to have it in the recipe the way it is on disk (so libgl is actually -lGL when being passed to GCC, for example, so use libGL.)